usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Usecase WebAPI
+ Scenario: Device & Hardware/BrowserState Test
+    When launch "usecase-webapi-xwalk-tests"
+     And I go to "/samples/BrowserState/index.html"
+     And I open wifi
+     And I wait 20 seconds
```

Comment:
WIFI connect to  internet need some times, wait 20 seconds to ensure that the connection is successful.